java/ql/integration-tests/all-platforms/java/buildless-inherit-trust-store/server.py

```diff
@@ -0,0 +1,12 @@
+from http.server import HTTPServer, SimpleHTTPRequestHandler
+import ssl
+
+
+httpd = HTTPServer(('localhost', 4443), SimpleHTTPRequestHandler)
```

Comment:
Minor: perhaps worth noting in a comment that `SimpleHTTPRequestHandler` serves the contents of the working directory and makes the repository server work. Since I am not a Python expert, I had to do some googling to figure out that this makes the whole thing work.

Reply:
ok, will follow up with a comment